README.md

```diff
@@ -195,4 +215,58 @@ garm-cli pool update --extra-specs='{"subnet_id":"subnet-0e7a29d5cf6e54789"}' <P
 
 You can also set a spec when creating a new pool, using the same flag.
 
-Workers in that pool will be created taking into account the specs you set on the pool.
\ No newline at end of file
+Workers in that pool will be created taking into account the specs you set on the pool.
+
+#### Supported Volume Parameters for Garm AWS Provider
+
+**NOTE**: The EBS Volume attached to the runner is configured to be deleted on termination and is set to have the `device name` set as `/dev/sda`.
+
+- `iops`
+
+    **Description**: Specifies the number of IOPS (Input/Output Operations Per Second) provisioned for the volume.
+    **Usage**:
+        **Required** for `io1` and `io2` volumes.
+        Optional for `gp3` volumes, with a default of 3,000 IOPS.
+        Not applicable for `gp2`, `st1`, `sc1`, or `standard` volumes.
+    **Valid Ranges**:
+        `gp3`: 3,000 - 16,000 IOPS
+        `io1`: 100 - 64,000 IOPS
+        `io2`: 100 - 256,000 IOPS (up to 32,000 IOPS on non-Nitro instances)
+    **Notes**:
+        For `gp2`, IOPS represents baseline performance and burst credit accumulation.
+
+- `throughput`
+
+    **Description**: Specifies the throughput (MiB/s) for the volume.
+    **Usage**:
+        **Valid only for `gp3` volumes**.
+        Not applicable for `gp2`, `io1`, `io2`, `st1`, `sc1`, or `standard` volumes.
+    **Valid Range**: 125 - 1,000 MiB/s
+
+- `volume_size`
+
+    **Description**: Specifies the size of the volume in GiB.
+    **Usage**:
+        Required unless a snapshot ID is provided.
+        Must be equal to or larger than the snapshot size if specified.
+    **Valid Ranges by Volume Type**:
```

Comment:
JSON schema allows enforcing choice, max, min. Things of this sort. You can add it to the `extraSpecs` type.

Reply:
For the `Iops` and the `VolumeSize`, I don't think we cannot do this directly through jsonschema, as it depends on the `VolumeType` parameter as well what the minimum/maximum will be. For now I have updated the `Throughput` to enforce minimum/maximum and the `VolumeType` enum of possible values